.gitmodules

```diff
@@ -28,3 +28,9 @@
 [submodule "tt_metal/third_party/tt_llk_blackhole"]
 	path = tt_metal/third_party/tt_llk_blackhole
 	url = https://github.com/tenstorrent/tt-llk-bh.git
+[submodule "tokenizers-cpp"]
+	path = tt-train/3rd_party/tokenizers-cpp
+	url = git@github.com:mlc-ai/tokenizers-cpp.git
+[submodule "3rd_party/wandb-cpp"]
+	path = tt-train/3rd_party/wandb-cpp
+	url = https://github.com/yhisaki/wandb-cpp
```

Comment:
Can we not get these from CPM, and treat them as external packages?

Reply:
Unfortunately it didn't work in other way because of the additional dependencies I wasn't able to make it work as CPM.
Also tokenizers-cpp was meant to be used as sub module (you can read their readme)

Reply:
I mean when I added this libraries I tried CPM/FetchContent first and it didn't work.

Reply:
why id tokenizers-cpp an ssh submodule when all the others are https?

This is causing problem in my local test env.